fighters/gamewatch/src/status.rs

```diff
@@ -18,7 +18,7 @@ mod landing_attack_air;
 
 pub fn install() {
     // smashline::install_agent_init_callbacks!(gamewatch_init);
-    landing_attack_air::install();
```

Comment:
Why is this not being installed?

Reply:
I'll let Askew answer but it looks like this was preventing edge slipoffs?

Reply:
Does essentially nothing now that dair's edge cancel is fixed

Reply:
Ah, I thought that was what fixed it lol